rust/datafusion/src/execution/context.rs

```diff
@@ -17,9 +17,11 @@
 
 //! ExecutionContext contains methods for registering data sources and executing queries
 
+use std::cell::RefCell;
 use std::collections::{HashMap, HashSet};
 use std::fs;
 use std::path::Path;
+use std::rc::Rc;
```

Comment:
As mentioned by @jorgecarleitao  -- I suggest using `Arc` in this file unless there is a good reason to use `Rc`